test/test_site.rb

```diff
@@ -14,14 +14,20 @@ class TestSite < JekyllUnitTest
 
     should "have an array for plugins if passed as a string" do
       site = Site.new(site_configuration({ "plugins_dir" => "/tmp/plugins" }))
-      assert_equal ["/tmp/plugins"], site.plugins
+      array = Utils::Platforms.really_windows? ? ["C:/tmp/plugins"] : ["/tmp/plugins"]
```

Comment:
the more I think about this, I wonder if this is a bug or intended..
when I set, `plugins_dir => "/apps/plugins"`
I should get a relative path to `plugins` dir inside my site, or an absolute_path to `plugins` dir **with** my site.